projects/js-packages/publicize-components/src/social-store/selectors/social-notes.ts

```diff
@@ -0,0 +1,37 @@
+import { store as coreStore } from '@wordpress/core-data';
+import { createRegistrySelector } from '@wordpress/data';
+import { getSocialScriptData } from '../../utils';
+import { SOCIAL_NOTES_CONFIG_KEY, SOCIAL_NOTES_ENABLED_KEY } from '../constants';
+
+/**
+ * Returns if Social Notes are enabled for the current site.
+ */
+export const getSocialNotesEnabled = createRegistrySelector( select => () => {
```

Comment:
Likewise here, should it be `isSocialNotesEnabled`?

Reply:
Fixed in 44f8c61, thanks!